hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -448,6 +448,10 @@ protected boolean isLob(int sqlTypeCode) {
 		};
 	}
 
+	protected boolean isUnsigned(String columnTypeName) {
+		return columnTypeName.contains( "UNSIGNED" );
+	}
```

Comment:
`unsigned` types are not, AFAIK, a standard or common feature of SQL databases, so this implementation doesn't belong as the default implementation on `Dialect`.